source/includes/steps-configure-custom-function-auth-realm-ui.yaml

```
@@ -51,36 +51,42 @@ content: |
      :alt: The function selection dropdown in the Realm UI.
      :width: 750px
      :lightbox:
-  
+
   .. example::
-     
+
      An application implements a Custom Function authentication provider
      that stores user data in the ``app.users`` MongoDB collection. The
      app lets users log in by specifying their ``username`` but does not
      require a password or any other type of authentication.
-     
-     The appication's authentication function queries the ``users``
+
+     The application's authentication function queries the ``app.users``
      collection for an existing user with the specified username. If the
      user already exists, the function returns their stored ``id`` value. If
      the user does not exist, the function stores a new user document in
      the collection and returns that document's ``id`` value.
-     
+
+     .. note::
+       The collection ``app.users`` is not created by Realm. You can use
+       your custom function with an external authentication system and/or
+       use data that you store in a MongoDB collection to identify and
```

```suggestion
       data that you store in a MongoDB collection to identify and
```
no need for an extra verb here!

---

Suggestion: I'm not sure where might be the best place for this, but [custom user data](https://docs.mongodb.com/realm/users/enable-custom-user-data/) is a pretty natural way to manage per-user data like this. Consider adding a link if you can find a way to bring it up organically!